src/components/Timetable.jsx

```diff
@@ -169,7 +169,7 @@ class Timetable extends React.Component {
     // Create days
     let days = []
     const selectedDay = weekdayNum(this.props.viewDate)
-    for (let i = 0; i < (this.props.days7 ? 7 : 5); i++) {
+    for (let i = 0; i < (this.props.days7 || isScreenSmall(this.props.screenSize) ? 7 : 5); i++) {
```

Comment:
Could you please move the resulting number to a separate assignment, e.g. `const dayCount`? Aside of being a little bit faster (since you don't need to reevaluate whole condition), it will be definitely readable.
Also, add comment about explicit small screen behaviour: Since we show just one day on mobile, always display the whole week to prevent hidden timetable on weekend.